### DISCUSSION TITLE
建议

### DISCUSSION THREAD

**Discussion by yihangyingzi** [share121/inter-knot#5117] [其他]
能加个帖子转发功能吗？有时候看到别人老发差不多的求助贴，又懒得一个个回。
或者加个置顶功能，把一些教程贴置顶。<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >

**Comment by KELERSAMA** [comment]
#+帖子编号 
例如
#2106 
#4497 
#4628

**Comment by yihangyingzi** [comment]
#5117
